ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/transceiver-qsfp-default-mode.j2

```diff
@@ -4,8 +4,8 @@
  that can be found in the LICENSE file.
 #}
 {# eos - transceiver qsfp default mode #}
-{# cannot be disabled in current EOS releases #}
-{% if not generate_default_config is arista.avd.defined(false) %}
+{# default qsfp mode is 40G for Tomahawk and Trident based devices starting from EOS 4.26.0F #}
+{% if not transceiver_qsfp_default_channelized is arista.avd.defined(false) %}
```

Comment:
I think we need to have this with an `or` so people who already have `default_config: false` will not suddenly see a change. We have to wait for AVD 5.0.0 to make breaking changes.

Reply:
Logic updated to use arista.avd.default filter, and be non-breaking.